environment.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
   - appdirs
   - filelock
   - fprettify
+  - fortran-language-server
```

Comment:
[this](https://github.com/fortran-lang/fortls#fortls-vs-fortran-language-server) makes it sound like `fortls` supersedes `fortran-language-server` but I have not used the latter so not sure.

Reply:
The intent was to add the fortran-language-server to the development environment. Will change to fortls.

Reply:
Actually it seems the modern fortran plugin for Visual Studio Code does not work with fortls but needs fortran-language-server (which has the same `fortls` exe name). See

https://stackoverflow.com/questions/72153608/fortran-language-server-fortls-error-in-vscode-windows

Reply:
Had a quick look at the fortls repo, it seems there are autodiscovery issues. Pointing vscode to it in workspace settings has worked if I recall, but that does not seem like it should be necessary and maybe there are other problems.

Sorry to lead astray, will look deeper in coming weeks